src/styles/catalog.scss

```diff
@@ -0,0 +1,8 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  padding-top: 50px;
+}
```

Comment:
Al long as you are using grid, see the task:

> Catalog using grid layout
> Implement the layout using grid
>
> Gap between cards should be: horizontal - 48px, vertical - 46px
> Blocks should start from the left
> Media queries:
> There should be four columns starting at 1024px
> There should be three columns starting at 768px
> There should be two columns starting at 488px
> There should be one column below 488px

Reply:
In addition - if it is ok with the task, better to use flexible column sizes (in `fr`)
If you need columns to be exactly `200px` consider using a variable (because it is the same as card width)